eo-maven-plugin/src/main/java/org/eolang/maven/BinarizeMojo.java

```diff
@@ -75,42 +78,45 @@ public final class BinarizeMojo extends SafeMojo {
     @Override
     public void exec() throws IOException {
         new Moja<>(BinarizeParseMojo.class).copy(this).execute();
-        final Path dest = targetDir.toPath().resolve("Lib");
-        final ProcessBuilder builder = new ProcessBuilder("cargo", "build")
-            .directory(dest.toFile());
-        Logger.info(this, "Building rust project..");
-        final Process building = builder.start();
-        try {
-            building.waitFor();
-        } catch (final InterruptedException exception) {
-            Thread.currentThread().interrupt();
-            throw new BuildFailureException(
-                String.format(
-                    "Interrupted while building %s",
-                    dest.toAbsolutePath()
-                ),
-                exception
-            );
-        }
-        if (building.exitValue() != 0) {
-            Logger.error(this, "There was an error in compilation");
-            final ByteArrayOutputStream stdout = new ByteArrayOutputStream();
-            try (VerboseProcess process = new VerboseProcess(building)) {
-                new Unchecked<>(
-                    new LengthOf(
-                        new TeeInput(
-                            new InputOf(process.stdoutQuietly()),
-                            new OutputTo(stdout)
+        for (final File file: targetDir.toPath().resolve("Lib").toFile().listFiles()) {
+            if (file.isDirectory() && file.toPath().resolve("Cargo.toml").toFile().exists()) {
+                Logger.info(this, String.format("Building rust project.."));
+                final ProcessBuilder builder = new ProcessBuilder("cargo", "build")
+                    .directory(file);
+                final Process building = builder.start();
+                try {
+                    building.waitFor();
+                } catch (final InterruptedException exception) {
+                    Thread.currentThread().interrupt();
+                    throw new BuildFailureException(
+                        String.format(
+                            "Interrupted while building %s",
+                            file
+                        ),
+                        exception
+                    );
+                }
+                if (building.exitValue() != 0) {
+                    Logger.error(this, "There was an error in compilation");
+                    final ByteArrayOutputStream stdout = new ByteArrayOutputStream();
```

Comment:
@levBagryansky Same here. Looks like we can inline this variable. According to: https://www.yegor256.com/2015/09/01/redundant-variables-are-evil.html